CMakeLists.txt

```diff
@@ -98,7 +98,11 @@ else (WIN32)
     enable_language(ASM)
 
     # Ensure that awk is present
-    find_program(AWK awk)
+    if (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
```

Comment:
This change is not needed, there is awk on FreeBSD too. If you want to enable gawk as an option for people that don't want to have awk installed, then please add a second `find_program` just without `if`. The `find_program` looks for the target program only when the `AWK` is not set yet or it is set to `AWK-NOTFOUND`.

Reply:
In fact it was to avoid the risk of different output (in future) between bsd awk and gnu awk but I can revert it no problems.